SageMaker/01_introduction.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Also an intro here would be good, e.g.: Now we setup a helper function to easily deploy any Hugging Face model as an endpoint on AWS SageMaker.</p><p> </p><p>I would also follow the function code a bit more closer. E.g. the <code>deploy</code> method creates and endpoint and we return the <code>predictor</code> object that we can use to call the endpoint. We return that object for later use.</p><p> </p><p>At the end we should add a note to call <code>destroy_endpoint</code> when done with the inference. </p><p> </p><p>Don't know if this is an review bug but each sentence seems to be on a new line - I don't think that's necessary.</p><p> </p><p>When referring the the class I would write <code>HuggingFaceModel</code>. For links I would use Markdown links: e.g. [<a href="https://docs.aws.amazon.com/sagemaker/latest/dg/deploy-model.html" target="_blank">Amazon SageMaker Inference Recommender</a></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>